epaint/src/tessellator.rs

```diff
@@ -192,6 +192,117 @@ pub mod path {
 
     use super::*;
 
+    mod precomputed_quadrants {
+        use emath::{vec2, Vec2};
+
+        pub const QUADRANTS_0_5: [[Vec2; 3]; 4] = [
+            // quadrant 0: right bottom
+            [
+                vec2(1.000000, 0.000000),
+                vec2(0.707107, 0.707107),
+                vec2(-0.000000, 1.000000)
+            ],
+            // quadrant 1: left bottom
+            [
+                vec2(-0.000000, 1.000000),
+                vec2(-0.707107, 0.707107),
+                vec2(-1.000000, -0.000000),
+            ],
+            // quadrant 2: left top
+            [
+                vec2(-1.000000, -0.000000),
+                vec2(-0.707107, -0.707107),
+                vec2(0.000000, -1.000000),
+            ],
+            // quadrant 3: right top
+            [
+                vec2(0.000000, -1.000000),
+                vec2(0.707107, -0.707107),
+                vec2(1.000000, 0.000000),
+            ],
+        ];
+
+        pub const QUADRANTS_5_10: [[Vec2; 5]; 4] = [
+            // quadrant 0: right bottom
+            [
+                vec2(1.000000, 0.000000),
+                vec2(0.923880, 0.382683),
+                vec2(0.707107, 0.707107),
+                vec2(0.382683, 0.923880),
+                vec2(-0.000000, 1.000000),
+            ],
+            // quadrant 1: left bottom
+            [
+                vec2(-0.000000, 1.000000),
+                vec2(-0.382684, 0.923880),
+                vec2(-0.707107, 0.707107),
+                vec2(-0.923880, 0.382683),
+                vec2(-1.000000, -0.000000),
+            ],
+            // quadrant 2: left top
+            [
+                vec2(-1.000000, -0.000000),
+                vec2(-0.923880, -0.382683),
+                vec2(-0.707107, -0.707107),
+                vec2(-0.382684, -0.923880),
+                vec2(0.000000, -1.000000),
+            ],
+            // quadrant 3: right top
+            [
+                vec2(0.000000, -1.000000),
+                vec2(0.382684, -0.923879),
+                vec2(0.707107, -0.707107),
+                vec2(0.923880, -0.382683),
+                vec2(1.000000, 0.000000),
+            ],
```

Comment:
Instead of having the four quadrants in separate array, we could just have all the circle vertices in circular order, and just slice into it. That way we could reuse the same vertices for circles too! It can wait for a separate PR though if you want.